sources/metrics/tracking pixels/TrackingPixelsConnector_Ad.swift

```diff
@@ -323,7 +323,8 @@ extension TrackingPixels.Connector {
         case .mp4:
             /* Playback Cycle Detector */ do {
                 let result = adPlaybackCycleDetector.process(streamPlaying: state.rate.adRate.stream,
-                                                             isFinished: state.adTracker.isFinished)
+                                                             isFinished: state.adTracker.isFinished,
```

Comment:
What is the difference between `isCompleted` and `isFinished`?

Reply:
Updated naming for understanding